client/gutenberg/extensions/subscriptions/editor.js

```diff
@@ -0,0 +1,60 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+//import './editor.scss';
+import edit from './edit';
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import JetpackBlockType from 'gutenberg/extensions/presets/jetpack/utils/jetpack-block-type';
+
+const SubscriptionBlock = new JetpackBlockType( 'subscriptions', {
+	title: __( 'Subscriptions' ),
+
+	description: (
+		<Fragment>
```

Comment:
We don't need the `<Fragment />` here, since it's wrapping a single element.

Then with removing the `<Fragment />`, we can also remove the `<p>`.